day_10/task_2/pipe_maze.cpp

```diff
@@ -211,16 +210,22 @@ void AttributedMaze::set_tile_at(const MazePoint& point, const Tile& tile)
 
 bool AttributedMaze::is_in_maze(const MazePoint& point) const
 {
-    return point.first >= 0 and point.first < static_cast<int>(maze_[0].size()) and
-        point.second >= 0 and point.second < static_cast<int>(maze_.size());
+    return point.first >= 0 and point.first < static_cast<int>(maze_.size()) and point.second >= 0 and point.second < static_cast<int>(maze_[0].size());
 }
 
 MazePoint operator+(const MazePoint& lhs, const MazePoint& rhs)
 {
     return MazePoint{lhs.first + rhs.first, lhs.second + rhs.second};
 }
 
-std::vector<MazePoint> generate_all_surroudning_tiles_starting_from_left_top_clock_wise(const MazePoint& mid_loop_point)
+void AttributedMaze::removeInvalidMazePoints(std::vector<MazePoint>& tiles) {
```

Comment:
```suggestion
void AttributedMaze::remove_invalid_maze_points(std::vector<MazePoint>& tiles) {
```
Please change it to snake_case

Reply:
[optional]

This function could be also delivered as separate commit with FIX prefix with description of the reason

Reply:
Really good catch that you spot the need to add this function and you implemented it! ❤️

Reply:
thank you <3